selftests/pre_release/jobs/timesensitive.py

```diff
@@ -11,9 +11,11 @@
 
 CONFIG = {
     'run.test_runner': 'nrunner',
-    'nrun.references': [os.path.join(ROOT_DIR, 'selftests', 'unit'),
-                        os.path.join(ROOT_DIR, 'selftests', 'functional')],
-    'filter_by_tags': ['parallel:1'],
+    'run.references': [os.path.join(ROOT_DIR, 'selftests', 'unit'),
+                       os.path.join(ROOT_DIR, 'selftests', 'functional')],
+    # FIXME: when using the job API, there's no command to be registered as part
+    # of the section name
+    'None.filter_by_tags': ['parallel:1'],
```

Comment:
The last commit on this series is fixing this one, right? Maybe we could merge it.

Reply:
The problem is that there is one issue fixed and the other "fixed by a workaround".  I can omit the second change (the workaround), but IMO the fix later would be worse "documented" (and the issue left open for longer, which can impact on bisects).   What do you think?

Reply:
On this commit you add this:

```
     'None.filter_by_tags': ['parallel:1'],
```

and later you fix to 

```
     'filter_by_tags.tags': ['parallel:1'],
```

IMO, only the last one is needed here.

Reply:
Right, I also don't like to change the same thing in the same PR, but because of `$REASONS` above, I think this is an exception... basically if I were looking for a failure on the job API layer using that job script, I wouldn't get a false positive at that commit.  It's a small thing, with two ways to look at it.  Let me know if the current state on v2 is OK to you.

Reply:
@clebergnu I'm not sure if understood the "reasons". By "reasons", are you referring to the following line?

> The problem is that there is one issue fixed and the other "fixed by a workaround".

I didn't get it. But ok, go ahead, feel free to merge-it.

Reply:
> @clebergnu I'm not sure if understood the "reasons". By "reasons", are you referring to the following line?
> 
> > The problem is that there is one issue fixed and the other "fixed by a workaround".
> 

Yes, I meant that I don't feel it's right to fix the `s/nrun.references/run.references`, but leave the other broken.  A workaround, keeping both working, is a better option IMO.

> I didn't get it. But ok, go ahead, feel free to merge-it.

OK, v2 is still lacking a formal review though. Thanks!